### ISSUE TITLE
Ways 193161110 and 193858774 are orphaned in database.

### ISSUE THREAD

**Issue by rwarren2** [OpenHistoricalMap/ohm-website#58]
Currently  causes a 500 when attempting to render. Doesn't seem to break anything else.

**Comment by batpad** [comment]
Since the server came back up, I tried to read the planet files on planet.openhistoricalmap.org and parse that using osmosis but this has been consistently failing because of missing node references. 
I used osmium's check-ref tool to see how many nodes are missing and there are about 3220 nodes missing that are referenced by ways. This is the case of planet files going back to June, that's as far back as I checked.

Rob, are the planet files generated from incremental diffs or directly from the database? This occurs sometimes on OSM as well and we can query from the API directly but a OHM returns a 500 when trying to fetch a node that's missing in the planet file for example https://openhistoricalmap.org/node/88864801
